build/templates/session.py.mako

```diff
@@ -13,7 +13,8 @@ ${encoding_tag}
 
     module_name = config['module_name']
     c_function_prefix = config['c_function_prefix']
-    attributes = template_parameters['metadata'].attributes
+
```

Comment:
For correctness, consider doing this in NI-Modinst's template.

Reply:
Done